src/domain/participant/index.js

```diff
@@ -175,10 +195,13 @@ const destroyByName = async (name) => {
 
 const addEndpoint = async (name, payload) => {
   try {
+    logger.debug('adding endpoint', { name, payload })
     const participant = await ParticipantModel.getByName(name)
     participantExists(participant)
+    logger.debug('adding endpoint for participant', { participant, payload })
```

Comment:
maybe, `info` severity?
  From my perspective, to define what severity to use  we should answer the question: Do we need to have this information during debugging production only by using logs?  If `yes` - then use `info`

Reply:
done, here and whereever appropriate.